samples/samplebinary.js

```diff
@@ -1,106 +0,0 @@
-/*
```

Comment:
why is this file deleted? Also transpiled?

Reply:
Yes, all JS needs to be transpiled.